examples/Bickley_jet_shallow_water.jl

```diff
@@ -76,43 +69,51 @@ model = ShallowWaterModel(
 
 # ## Background state and perturbation
 #
-# The background velocity ``\overline{u}`` and free-surface ``\overline{\eta}`` are chosen to 
-# represent a geostrophically balanced Bickely jet with maximum speed of ``U`` and 
-# maximum free-surface deformation of ``Δη``, i.e.,
+# The background velocity ``ū`` and free-surface ``η̄`` are chosen to represent a 
```

Comment:
I agree that using the short version is better and thanks for suggesting this. But does the overline on the `\eta` look too far to the right?  I guess it's how it looks in the html that matters.

Reply:
hm... it looks fine on my font :)

<img width="150" alt="Screen Shot 2021-03-15 at 8 07 34 am" src="https://user-images.githubusercontent.com/7112768/111084237-85540800-8565-11eb-8120-7e894663c607.png">

Reply:
Mine looks a bit odd but again, this is only the code so don't think this is a problem, just wanted to mention it.
![Screenshot from 2021-03-14 17-12-54](https://user-images.githubusercontent.com/8239041/111084399-91e64b00-84e8-11eb-81c6-f8a6f184f8ea.png)

Reply:
Have a look at this https://cormullion.github.io/pages/2020-07-26-JuliaMono/

Reply:
Thanks and will try and get that working on my computer